common/src/main/java/com/wynntils/functions/InventoryFunctions.java

```diff
@@ -256,7 +258,9 @@ public NamedValue getValue(FunctionArguments arguments) {
             Optional<ShinyStat> shinyStatOpt = Models.Shiny.getShinyStat(itemStack);
             if (shinyStatOpt.isEmpty()) return NamedValue.EMPTY;
 
-            return new NamedValue(shinyStatOpt.get().name(), shinyStatOpt.get().value());
+            // FIXME: The function system can't handle longs, so we have to cast to int
```

Comment:
This irks me a bit. Are we planning to add longs to the function system? Can't we just remove the comment otherwise? Or are shinyStat values likely to overflow an int? If so, maybe we need to fix it. Otherwise, maybe it should return a int instead of a long.

Not a big thing, but it kind of bugs me.

Reply:
Ok, let's leave the fixme for now then. Maybe open a bug to track that we need to handle this properly in the future.